WordPress/src/main/java/org/wordpress/android/ui/prefs/ProfileInputDialogFragment.java

```diff
@@ -79,11 +91,8 @@ public Dialog onCreateDialog(Bundle savedInstanceState) {
         alertDialogBuilder.setCancelable(true)
                 .setPositiveButton(R.string.ok, new DialogInterface.OnClickListener() {
                     public void onClick(DialogInterface dialog, int id) {
-                        if (getActivity() instanceof Callback) {
-                            ((Callback) getActivity()).onSuccessfulInput(editText.getText().toString(), callbackId);
-                        } else {
-                            AppLog.e(AppLog.T.UTILS, getActivity().getClass().getName()
-                                    + " doesn't implement ProfileInputDialogFragment.Callback");
+                        if (mCallback != null) {
```

Comment:
Since this is the only place where `mCallback` is used, perhaps we can avoid holding a reference to `mCallback` and simply use this here:
```
if (getTargetFragment() instanceof Callback) {
     ((Callback) getTargetFragment()).onSuccessfulInput(editText.getText().toString(), callbackId);
}
```